htlcswitch/link.go

```diff
@@ -2823,21 +2823,7 @@ func (l *channelLink) processExitHop(pd *lnwallet.PaymentDescriptor,
 		invoiceHash, pd.Amount, pd.Timeout, int32(heightNow),
 		circuitKey, l.hodlQueue.ChanIn(), payload,
 	)
-
-	switch err {
-
-	// Cancel htlc if we don't have an invoice for it.
```

Comment:
not obvious to me why this can be removed? Add explanation to commit message perhaps?

Reply:
This is not removed, but moved to invoice registry. It will return a cancel resolution if the invoice is not found.

Reply:
Ah, I see it now. Thanks!